apps/src/aichat/redux/slice.ts

```diff
@@ -103,6 +105,10 @@ const aichatSlice = createSlice({
     setUserHasAichatAccess: (state, action: PayloadAction<boolean>) => {
       state.userHasAichatAccess = action.payload;
     },
+
+    setClientType(state, action: PayloadAction<AiChatClientType>) {
```

Comment:
I saw Sanchit mention we'd ideally not have this in Redux (but ok for now) -- any thoughts from this work on where it might make sense for this to fit eventually?

Also, maybe a comment here that in the meantime we really don't want to be using this (apart from this use case) as we want the client type to be managed via props?